components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -63,7 +124,15 @@ service Policies {
     option (chef.automate.api.policy).action = "delete";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}";
     option (chef.automate.api.iam.policy).action = "iam:policies:delete";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Update an existing policy
+
+  Very similar to create except the ID cannot be changed.
```

Comment:
Actually, there is more to say about updates--this is adapted from what was deleted in the .md file and should be included here:

"When updating a policy you must supply all of that policy's properties, not just the ones you wish to update.
Properties that you do not include are reset to empty values!

You can use this endpoint to modify _custom_ policies but not _Chef-managed_ policies.

If you just wish to update members on a policy (either custom or Chef-managed), see: Add policy members, Remove policy members, and Replace policy members"